docs/modules/ROOT/pages/how-tos/testing/integration/debugging.adoc

```diff
@@ -0,0 +1,44 @@
+= Debugging Integration Tests
+
+In the event of an integration test scenario failing, there are a number of steps to take in order to debug the issue before rerunning the test in question. These steps are outlined below.
+
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before attempting to debug the failed test.
+
+- You have access to the console in order to view the logs of the failed test.
+
+.Procedure
+
+. Identify the IntegrationTestScenario that has failed. This can be found in the console by navigating to 
+Applications/your_application_name/activity and select "Pipeline Runs" under "Activity By". Your view should be similar to the below image.
```

Comment:
Would it make sense to separate these steps out similar to how the verification steps have been listed for other integration test pages?

Reply:
Sure thing! Just to make sure I have it right, more like a single sentence per step?